src/app/components/action-button/index.html

```diff
@@ -31,4 +31,162 @@
+    </stache-row>
+    
+    <header><div class="sky-subsection-heading">When not to use action buttons</div></header>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Do not use a single action button by itself. Action buttons should not serve as a primary call-to-action in a page where many other actions are available to the user.
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/single-action-button-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Do not use actions buttons to present different methods for completing a task when the user has not initated the task yet. Action buttons should not compete against other content or actions for the user’s attention.
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-on-workspace-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Do not use actions buttons when the consequential differences between the choices are a small minority of the scale of the task, or when the choice would not be made at the beginning of the task. Use a different selection input and progressive disclosure to reveal additional content, if needed.
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-inside-long-dialogue-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <header><div class="sky-section-heading">Workflow placement and behaviors</div></header>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Use a set of action buttons inside a modal when no page already exists, or the remaining bulk of the task after the action button decision takes place inside a modal. This is common when creating a new record that only requires a standard modal during different versions of its 'Add' process.
+            </p>
+            <ul>
+                <li>Use a 'default' size modal for 2 action buttons and a 'largeModal' for 3 or more.</li>
+                <ul>
+                    <li>Center the action buttons inside the modal, wrapping to another row if needed.</li>
+                </ul>
+                <li>When the user selects a choice, dismiss this modal and open whatever container is appropriate for the remainder of the task.</li>
```

Comment:
It might be worth reviewing this with the team for feasibility, and then with Todd to see if it would even be a preferable option. Knowing nothing about what container is needed after the action button decision (and it might be different for different paths), we opted for a design where the modal containing the action buttons was "disposable".

If it would be possible to resize/transform/repopulate the modal with all of the subsequent interactions after choosing a path, that might be worth considering _in the cases where the appropriate modal size is the same as the one used to contain the action buttons_. I'm not convinced it's actually better, since there are some baseline benefits to the action button modal always behaving in a predictable and expected way.